README.md

```diff
@@ -133,16 +133,35 @@ Ubuntu 16.04     | no
 Ubuntu 18.04     | yes
 
 
-## Localization
+## Localization and translation
+
+Zonemaster comes with translation for the following
+languages
+
+* English (en, en_US.UTF-8)
+* French (fr, fr_FR.UTF-8)
+* Swedish (sv, sv_SE.UTF-8)
+* Danish (da, da_DK.UTF-8)
+
+The support of localization or translation is however not complete due to
+complexity in the implementation of `gettext` functions in different OSs.
+The table below documents in what OS the translation function works for
+Engine (i.e. the Zonemaster libraries), the CLI (i.e. the CLI interface
+to the Engine) and Backend (i.e. the RPCAPI interface to the Engine). If
+translation does not work properly in the Backend, it will not work in
+GUI either.
+
+Operating System | Engine | CLI | Backend
+---------------- | -------|-----|--------
+CentOS 7         | ?      |  ?  |   ?
+CentOS 8         | ?      |  ?  |   ?
+Debian 9         | ?      |  ?  |   ?
+Debian 10        | ?      |  ?  |   ?
+FreeBSD 11.3     | yes    | yes | no
+FreeBSD 12.1     | yes    | yes | yes
```

Comment:
AFAICT Engine is relying on undefined behavior in the l10n APIs we're using. It happens to work on Linux but on FreeBSD they're implemented differently. CLI was working around this bug in Engine all along, and now Backend is working around it too. IMHO we should either integrate the workaround in Engine or say something other than yes here.

Reply:
OK, a "no" for Engine for FreeBSD?

Reply:
Translation does work in some contexts, e.g. CLI, so a simple no isn't accurate either. I suggest "partial" with a link to an issue. I'm not sure if there is an issue to link to. Do you want me to create one?

Reply:
Updated.